### DISCUSSION TITLE
why not the policy in  133/terraform/9-csi-driver-iam.tf data.aws_iam_policy_document

### DISCUSSION THREAD

**Discussion by josepedropinto2022** [community/community#39874] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hi, why you don't create the policy instead using  this data "aws_iam_policy_document" ?

**Comment by ossanna16** [comment]
Hi @josepedropinto2022,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on AWS. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
